components/cache/cache_invalidation.rst

```diff
@@ -10,20 +10,22 @@ change in the state of your model. The most basic kind of invalidation is direct
 items deletion. But when the state of a primary resource has spread accross
 several cached items, keeping them in sync can be difficult.
 
-The Symfony Cache component provides two mechanisms to help solve this problem:
+The Symfony Cache component provides two mechanisms to help solving this problem:
 
-* Tags based invalidation for managing data dependencies;
-* Expiration based invalidation for time related dependencies.
+* :ref:`Tags based invalidation <cache-component-tags>` for managing data dependencies;
+* :ref:`Expiration based invalidation <cache-component-expiration>` for time related dependencies.
 
-.. versionadded:: 3.2
-    Tags based invalidation was introduced in Symfony 3.2.
+.. _cache-component-tags:
 
 Using Cache Tags
 ----------------
 
+.. versionadded:: 3.2
+    Tags based invalidation was introduced in Symfony 3.2.
+
 To benefit from tags based invalidation, you need to attach the proper tags to
-each cached items. Each tag is a plain string identifier that you can use at any
-time to trigger the removal of all items that had this tag attached to them.
+each cached item. Each tag is a plain string identifier that you can use at any
+time to trigger the removal of all items that have this tag attached to them.
```

Comment:
`all items that have this tag attached to them.` -> `all items associated with this tag.` ?